docs-gb/getting-started/kubernetes-installation/README.md

```diff
@@ -2,9 +2,7 @@
 
 ## Prerequisites
 
-### A running Kubernetes Cluster
-
-You will need a running Kubernetes cluster or can create a local KinD one for testing via [Ansible](ansible.md).
+* Ensure that the version of the Kubernetes cluster is v1.27 or later. You cam create a Kubernetes cluster on local KinD for testing with [Ansible](ansible.md). 
```

Comment:
a minor spelling error: `You cam create` should become `You can create`

Reply:
Thank you! Just committed the fix.